conda_rpms/install.py

```diff
@@ -399,6 +404,195 @@ def symlink_conda(prefix, root_dir):
+    Modified version of conda/gateways/disk/permissions.py:make_executable at
+    be8c08c083f4d5e05b06bd2689d2cd0d410c2ffe.
+
+    Modifications included:
+        * changed lstat -> os.lstat
+        * changed chmod -> os.chmod
+        * changed log.trace -> log.info (log.trace not supported)
+
+    """
+    if isfile(path):
+        mode = os.lstat(path).st_mode
+        log.info('chmod +x %s', path)
+        os.chmod(path, S_IMODE(mode) | S_IXUSR | S_IXGRP | S_IXOTH)
+    else:
+        log.error("Cannot make path '%s' executable", path)
+
+
+def replace_long_shebang(data):
+    """
+    Modified version of conda/core/portability.py:replace_long_shebang
+    at be8c08c083f4d5e05b06bd2689d2cd0d410c2ffe.
+
+    Modifications included:
+        * removed mode check for non-binary shebang
+        * brought `SHEBANG_REGEX` into the function
+
+    """
+    # three capture groups: whole_shebang, executable, options
+    SHEBANG_REGEX = (br'^(#!'  # pretty much the whole match string
+                     br'(?:[ ]*)'  # allow spaces between #! and beginning of the executable path
+                     br'(/(?:\\ |[^ \n\r\t])*)'  # the executable is the next text block without an escaped space or non-space whitespace character  # NOQA
+                     br'(.*)'  # the rest of the line can contain option flags
+                     br')$')  # end whole_shebang group
+    shebang_match = re.match(SHEBANG_REGEX, data, re.MULTILINE)
+    if shebang_match:
+        whole_shebang, executable, options = shebang_match.groups()
+        if len(whole_shebang) > 127:
+            executable_name = executable.decode('utf-8').split('/')[-1]
+            new_shebang = '#!/usr/bin/env %s%s' % (
+            executable_name, options.decode('utf-8'))
+            data = data.replace(whole_shebang, new_shebang.encode('utf-8'))
+    return data
+
+
+def create_python_entry_point(target_full_path, python_full_path, module, func):
+    """
+    Modified version of conda/gateways/disk/create.py:create_python_entry_point
+    at be8c08c083f4d5e05b06bd2689d2cd0d410c2ffe.
+
+    Modifications included:
+        * Removed check that the entry point already exists
+
+    """
+    import_name = func.split('.')[0]
+    pyscript = python_entry_point_template % {
+        'module': module,
+        'func': func,
+        'import_name': import_name,
+    }
+
+    if python_full_path is not None:
+        shebang = '#!%s\n' % python_full_path
+        if hasattr(shebang, 'encode'):
+            shebang = shebang.encode()
+
+        shebang = replace_long_shebang(shebang)
+
+        if hasattr(shebang, 'decode'):
+            shebang = shebang.decode()
+
+    else:
+        shebang = None
+
+    with open(target_full_path, str('w')) as fo:
+        if shebang is not None:
+            fo.write(shebang)
+        fo.write(pyscript)
+
+    if shebang is not None:
+        make_executable(target_full_path)
+
+    return target_full_path
+
+
+def get_python_version(prefix):
```

Comment:
@lbdreyer Is this your own function or is it sourced from `conda`?

Reply:
Mine, if it was sourced from conda I stated that in the function docstring

Reply:
Cool, thanks!